examples/cucumber-js/README.md

```diff
@@ -33,6 +33,25 @@ $ npm i @cucumber/cucumber --save-dev
 }
 ```
 
+## CLI Options
```

Comment:
A small description can be added so that users know that these are the only options supported for cucumber. Also, IMO it would be great if we also let them know why we are not supporting the rest of the CLI alternatives. Let me about your options

Reply:
Won't it be clear if it is mentioned under Nightwatch-Cucumber docs section that only these options are supported? I don't think so we should add a reason for not supporting a CLI option in the doc